tests/ui/parser/break_outside_loop.sol

```diff
@@ -0,0 +1,3 @@
+function f() {
+    break; //~^ERROR: `break` outside of a loop
```

Comment:
```suggestion
    break; //~ ERROR: `break` outside of a loop

    { break; } //~ ERROR: `break` outside of a loop
```

Please also add the same function inside of a `contract C`

Reply:
Check the update smart contract and let me know if that looks okay.